modules/ROOT/pages/basic-alerts.adoc

```diff
@@ -27,17 +27,115 @@ To create a basic alert:
 . Click the *Alerts* tab.
 . Click the *+ New Alerts* button.
 . Click *Basic alert*.
-. In the *Add basic alert* window, enter the information required to create your basic alert. 
+. In the *Add basic alert* window, enter the information required to create your basic alert:
+
+* *Name*
++
+Unique and meaningful name for the alert
++
+The *Name* is limited to 256 characters.
+* *Severity level*
++
+Severity of the alert: *Critical*, *Warning*, or *Info*
+See <<severity>>.
++
+* *Source type*
++
+The source of the alert: applications or servers
++
+* *Environment name*
++
+The name of the environment where the resource is deployed
++
+* *Resource name* or *Server name*
++
+The application or server to apply the alert to
+* *Condition*
 +
-Note that requirements change depending on the source type you select.
-. Click *Save*.
+The condition that triggers the alert
++
+See <<alert-conditions>>.
+* *Subject*
++
+The subject of the alert email to send when the alert is triggered
++
+// The subject can include variables.
+// See <<alert-variables>>.
+* *Message*
++
+The body text for the email to send when the alert is triggered
+// The message body can include variables.
+// See <<alert-variables>>.
+* *Recipients*
++
+The recipients of the alert email
++
+You can select up to 20 platform user IDs and enter up to 20 email addresses.
+. Click *Create*.
 
 image::basic-alerts-config.png[Configure a Basic Alert, 50%, 50%]
 
-=== Basic Alert Severity
+[[severity]]
+=== Severity
 
 The following levels of severity are available for basic alerts:
 
 include::partial$include-alert-severity.adoc[leveloffset=+1]
 
+[[alert-conditions]]
+=== Alert Conditions
+
+For basic alerts, alert conditions consist of a metric, an operator, and a period of time that the condition is met.
+
+==== Metrics
+
+The following metrics are available if your alert is created for an application:
+
+Message count::
+The total number of Mule messages that are processed successfully
+
+Message error count::
+The total number of error messages. This includes 400- and 500- errors.
```

Comment:
I do not think there is any relationship with 400 and 500 errors. @vailaya do we have a better definition for this metric? Can we just say its the number of mule messages which failed to be processed?

Reply:
@ejaz-vel, @sarathecoubian  The mule runtime team described the error count as follows (https://www.mulesoft.org/jira/browse/SE-18424):

the message error count metric is incremented with every exception thrown in a flow.

Reply:
```suggestion
The number of Mule messages that return an exception in a flow.
```

Reply:
Sounds good